examples/advanced_training/plot_exca_config.py

```diff
@@ -0,0 +1,409 @@
+# type: ignore
+"""Experiment configuration with Pydantic and Exca
+==================================================
+
+This example shows how to use the ``pydantic`` and ``exca`` libraries
+to configure and run EEG experiments with Braindecode.
+
+**Pydantic** is a library for data validation and settings management
+using Python type annotations. It allows defining structured configurations that can be
+validated and serialized easily.
+
+**Exca** builds on top of Pydantic, and allows you to seamlessly EXecute experiments
+and CAche their results.
+
+Braindecode implements a Pydantic configuration for each of its models in
+``braindecode.models.config``.
+In this example, we will use these configurations to define an experiment that
+trains and evaluates different models on a motor-imagery dataset using Exca.
+
+.. contents:: This example covers:
+   :local:
+   :depth: 2
+"""
+
+# Authors: Pierre Guetschel
+#
+# License: BSD (3-clause)
+
+#####################################################################
+# Creating the experiment configurations
+# -------------------------------------
+#
+# We will start by defining the configurations needed for our experiment using Pydantic and Exca.
+#
+# Dataset configs
+# ~~~~~~~~~~~~~~~~~~
+#
+# Our first configuration class is related to the data. It will allow us to load and prepare the dataset.
+import exca
+import pydantic
+
+from braindecode import EEGClassifier
+from braindecode.datasets import MOABBDataset
+from braindecode.preprocessing import create_windows_from_events
+
+
```

Comment:
```suggestion
import warnings

import exca
import pydantic

from braindecode import EEGClassifier
from braindecode.datasets import MOABBDataset
from braindecode.preprocessing import create_windows_from_events

warnings.simplefilter("ignore")


```
try to remove the warnings